docs/website/content/docs/installation.md

```diff
@@ -10,15 +10,17 @@ toc: true
 
 Sysl is a CLI (Command Line Interface) that executes with the `sysl` command.
 
+Install it with
+
     GO111MODULE=on go get github.com/anz-bank/sysl/cmd@v0.4.0
 
 ## Prerequisites
 
-- [Go](https://golang.org)
-- [PlantUML](https://hub.docker.com/r/plantuml/plantuml-server/) server for diagram generation for use if using the [external service](http://www.plantuml.com/plantuml/) is not appropriate
-
-## Setting PlantUML Environment variable
-
-In order to be able to generate diagrams the `SYSL_PLANTUML` Environment variable needs to be set
+- [Go 1.13](https://golang.org/doc/install)
+- PlantUML environment variable for diagram generation:
+  `export SYSL_PLANTUML=http://www.plantuml.com/plantuml`
 
-    export SYSL_PLANTUML=http://www.plantuml.com/plantuml
+If the unauthenticated, external [PlantUML
```

Comment:
I don't think "unauthenticated" is the right word here. I would just say something like:

If the external PlantUML service is not suitable for your use case, you can run a local instance of the PlantUML server using the docker image and point the `SYSL_PLANTUML` environment variable to that instance.

Reply:
Done. Thanks for your clarity.